tests/cypress/e2e/hedy_page/editor_box.cy.js

```diff
@@ -3,18 +3,12 @@ import {goToHedyPage} from "../tools/navigation/nav";
 describe('Is able to type in the editor box', () => {
     it('Passes', () => {
       goToHedyPage();
-
-      cy.get('#editor > .ace_scroller > .ace_content').type('\nask What is your name\necho hello');
-      cy.get('#editor > .ace_scroller > .ace_content .ace_line').each((element, index) => {
-        if(index == 0) {
-          cy.get(element).should('have.text', 'print hello world!');
-        }
-        if(index == 1) {
-          cy.get(element).should('have.text', 'ask What is your name');
-        }
-        if(index == 2) {
-          cy.get(element).should('have.text', 'echo hello');
-        }
-      })
+      // click on textaread to get focus

```

Comment:
I think this works nicely! But I think it only tests with English language setting. It would be nice to also test it for a few other languages, can that be done?